cmd/config-manager/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Build Stage
+ARG GO_VERSION=1.19
+
+FROM golang:${GO_VERSION}-buster as build
```

Comment:
Please use golang:${GO_VERSION}-bullseye for building like the rest of the plugins/binaries. That'll give us up to date golang toolchain.